UnitedIncome/components
# App 5043 radio button card ts

### Description
* Modify some types in `components` to take advantage of event specific type definitions provided by react.
* Update `RadioButtonCard`'s test to use react-testing-library

### Testing Instructions
Ensure `RadioButtonCard` continues to work as expected.

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-5043)
